ops/terraform/modules/resources/templates/pipeline_server.tpl

```diff
@@ -8,12 +8,6 @@ exec > >(
 	done
 )
 
-# Extend gold image defined root partition with all available free space
-sudo growpart /dev/nvme0n1 2
-sudo pvresize /dev/nvme0n1p2
-sudo lvextend -l +100%FREE /dev/VolGroup00/rootVol
-sudo xfs_growfs /
-
```

Comment:
No longer needed in AL2, root volume is expanded automagically upon defining extra space.